lava_test_plans/testcases/master/template-android-xts.jinja2

```diff
@@ -0,0 +1,46 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% if xts_test_url is defined %}
+{% set xts_test_params = xts_test_params|default("xts-not-work") %}
+{## the test_name is a part of the job name ##}
+{% set test_name = test_name|default("xts") %}
+{## the test_suite_name is a part of the path of the lava result ##}
+{% set xts_test_suite_name = xts_test_suite_name|default(test_name) %}
+{% set xts_test_path = xts_test_path|default("xts-path") %}
+{% set xts_pkg_name = xts_pkg_name|default("android-xts.zip") %}
+{% set xts_expect_reboot = xts_expect_reboot|default(false) %}
+{% set xts_result_format = xts_result_format|default("aggregated") %}
+{% set xts_test_definition_path = xts_test_definition_path|default("automated/android/noninteractive-tradefed/tradefed.yaml") %}
+{% set xts_metadata_prefix = xts_metadata_prefix|default('xts') %}
+{% set xts_version = xts_version|default("xts-version-unknown") %}
+{% else %}
+{% set test_name = test_name|default("TEST_XTS_URL-not-defined") %}
+{% endif %}
+
+{% block metadata %}
+{% if xts_test_url is defined %}
+  {{ super() }}
+  {{xts_metadata_prefix}}-url: "{{ xts_test_url }}/android-cts.zip"
+  {{xts_metadata_prefix}}-manifest: "{{ xts_test_url }}/pinned-manifest.xml"
+  {{xts_metadata_prefix}}-version: "{{ xts_version }}"
+{% endif %}
+{% endblock metadata %}
+
+{% block test_target %}
+{% if TEST_XTS_URL is defined %}
+  {{ super() }}
+    - repository: {{TEST_DEFINITIONS_REPOSITORY}}
+      from: git
+      path: {{xts_test_definition_path}}
+      params:
+        TEST_PARAMS: {% block xts_test_params %}{{xts_test_params}}{% endblock xts_test_params %}
+        TEST_URL: "{{TEST_XTS_URL}}/{{xts_pkg_name}}"
```

Comment:
should be test_xts_url and not TEST_XTS_URL

Reply:
thanks, updated